tests/integration/05.Dfw.Tests.ps1

```diff
@@ -90,6 +90,9 @@ Describe "DFW" {
         $script:testvm1 = new-vm -name $testVMName1 @vmsplat
         $script:testvm2 = new-vm -name $testVMName2 @vmsplat
         $script:testvm3 = new-vm -name $testVMName3 @vmsplat
+        
+        Get-VM $testVMName1 | New-NetworkAdapter -StartConnected -Portgroup ($testls | Get-NsxBackingPortGroup).name | out-null
```

Comment:
can have more than one backing PG for an LS (will have in our standard test environment too) so this needs to get the one in the correct cluster.